test/runtests.jl

```diff
@@ -438,9 +438,9 @@ cd(@__DIR__) do
     # o_ts.verbose = true # set to true to show all timings when successful
     Test.print_test_results(o_ts, 1)
     if !o_ts.anynonpass
-        println("    \033[32;1mSUCCESS\033[0m")
+        printstyled("    SUCCESS\n"; bold=true, color=2)
```

Comment:
Why 2 and not `:green`?

Reply:
Because I thought I tried `:green` and I got a weird colour, but I just realised I had tried `:greeen`.